src/Components/Web/src/Forms/InputRadioGroup.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using Microsoft.AspNetCore.Components.Rendering;
+
+namespace Microsoft.AspNetCore.Components.Forms
+{
+    /// <summary>
+    /// Groups child <see cref="InputRadio{TValue}"/> components.
+    /// </summary>
+    public class InputRadioGroup : ComponentBase
+    {
+        private string? _name;
+
+        /// <summary>
+        /// Gets or sets the child content to be rendering inside the <see cref="InputRadioGroup"/>.
+        /// </summary>
+        [Parameter] public RenderFragment? ChildContent { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the group.
+        /// </summary>
+        [Parameter] public string? Name { get; set; }
+
+        /// <inheritdoc />
+        protected override void OnParametersSet()
+        {
+            _name ??= !string.IsNullOrWhiteSpace(Name) ? Name : Guid.NewGuid().ToString("N");
```

Comment:
Is it useful to configure the `Name` property with the generated guid?

Reply:
Would https://docs.microsoft.com/en-us/aspnet/core/blazor/components/?view=aspnetcore-3.1#dont-create-components-that-write-to-their-own-parameter-properties be a concern if we did that?